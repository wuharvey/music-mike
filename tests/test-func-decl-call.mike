
def Name = {Printint(10); Printstr("hello"); a = 5 + 10; a;};

Printint(Name());

