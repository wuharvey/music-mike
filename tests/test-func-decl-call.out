10
hello
15
